src/lib/commands/config.ts

```diff
@@ -73,14 +77,18 @@ export const registerCommands = (context: ExtensionContext) => {
       COMMANDS.CONFIG.SELECT_DEFAULT_ACCOUNT,
       async () => {
         const config = getConfig();
+        // @ts-expect-error TODO: Fix this when updating local-dev-lib
         if (config && config.portals) {
```

Comment:
Are there any utils we could leverage from LDL to get around some of these issues? Would `getConfigAccounts` help here?